cmd/thanosbench/stress.go

```diff
@@ -0,0 +1,114 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"math"
+	"math/rand"
+
+	"time"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/oklog/run"
+	"github.com/pkg/errors"
+	"github.com/prometheus/prometheus/pkg/labels"
+	"github.com/thanos-io/thanos/pkg/store/storepb"
+	"golang.org/x/sync/errgroup"
+	"google.golang.org/grpc"
+	"gopkg.in/alecthomas/kingpin.v2"
+)
+
+func registerStress(m map[string]setupFunc, app *kingpin.Application) {
+	cmd := app.Command("stress", "Stress tests a remote StoreAPI.")
+	workers := cmd.Flag("workers.num", "Number of go routines for stress testing.").Required().Int()
```

Comment:
Maybe better to be consistent on the name with other sub-commands. Is it better change to `workers`?

Reply:
Good point, changed it :+1: